server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java

```diff
@@ -210,12 +137,16 @@ public Issue getByKey(String key) {
     return issueIndex.getByKey(key);
   }
 
+  // TODO Either use IssueFinder or remove it
+  @Deprecated
   IssueDto getByKeyForUpdate(DbSession session, String key) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)